### PR TITLE
Modules topics

### PR DESCRIPTION
**Pull request by diana-lakatos**
Concept topic and tutorial series for Modules. Please review until approval, but don't merge until https://github.com/mdyd-dev/nearme-documentation/pull/243 has been merged. Thank you!

### CODE REVIEW

File: marketplace_builder/views/pages/how-to/modules/creating-module.liquid [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+            └── views
+                └── pages
+                    └── admin.html.liquid
+```
+
+
+
+## Live example and source code
+
+You can download the directory structure and all code included in this tutorial [here](https://github.com/mdyd-dev/nearme-documentation/tree/master/marketplace_builder/assets/files/tutorials/modules/admincms.zip) as a zip file.
```
Comment (pavelloz) [inline marketplace_builder/views/pages/how-to/modules/creating-module.liquid]: Can we put it into example marketplace and link to source code + working page like we did with couple other guides?

Also i would move zip file there or remove it completely - im pretty sure that in the next 3 code changes zip will become outdated ;)

https://github.com/mdyd-dev/marketplace-nearme-example - this is the repo where we can show off stuff - i can go through tutorial and do it - additional user test wont hurt.

Comment (diana-lakatos) [reply marketplace_builder/views/pages/how-to/modules/creating-module.liquid]: @pavelloz That's a great idea! Thank you for offering to go through the tutorial and do it.

**Review by ignacy — APPROVED** [review]
LGTM, thanks! :shipit:

File: marketplace_builder/views/pages/how-to/modules/creating-module.liquid (hunk 1 above)
Comment (pavelloz) [reply marketplace_builder/views/pages/how-to/modules/creating-module.liquid]: Ok, let me do this then.
I should make it this week so the next week update/accnouncement could include it

File: marketplace_builder/views/pages/how-to/modules/creating-module.liquid [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+Create the page `admincms/public/views/pages/admin.html.liquid`:
+
+```liquid
+{% raw %}
+---
+slug: admincms
+layout_name: 1col
+---
+
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mini.css/3.0.0/mini-default.min.css">
```
Comment (pavelloz) [inline marketplace_builder/views/pages/how-to/modules/creating-module.liquid]: also, when moving it to nearme-examples ill remove it as there is already bootstrap loaded for prettier styling

**Comment by pavelloz** [comment]
I made couple changes:
1) formatting
2) in code necessary for live example to be correct
3) removed some code - docs layout is narrow + there were some unnecessary line breaks
4) removed zip file + linked to source code on github 

Live example lives here: https://nearme-example.staging-oregon.near-me.com/admincms
Source code: https://github.com/mdyd-dev/marketplace-nearme-example/tree/master/marketplace_builder/modules/admincms

**Comment by pavelloz** [comment]
BTW. I havent checked if everything works on PP side, i was only doing the code part

**Comment by diana-lakatos** [comment]
@pavelloz 
> BTW. I havent checked if everything works on PP side, i was only doing the code part

It's OK, I have checked the whole process on the PP.

**Pull request merged**